PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/UseOfBlockOnAsyncClientsCheck.java

```diff
@@ -0,0 +1,165 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.JavaElementVisitor;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiClassType;
+import com.intellij.psi.PsiExpression;
+import com.intellij.psi.PsiMethodCallExpression;
+
+import com.intellij.psi.PsiReferenceExpression;
+import com.intellij.psi.PsiType;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Inspection tool to check for the use of block() method on async clients in Azure SDK.
+ */
+public class UseOfBlockOnAsyncClientsCheck extends LocalInspectionTool {
+
+    /**
+     * This method is used to get the visitor for the inspection tool.
+     * The visitor is used to check for the use of block() method on async clients in Azure SDK.
+     * The visitor checks if the method call is a block() method call on a reactive type
```

Comment:
Same here - all documentation should refer to "blocking calls" instead of mentioning just `block()` method.